internal/corazawaf/transaction.go

```diff
@@ -561,29 +561,27 @@ func (tx *Transaction) GetField(rv ruleVariableParams) []types.MatchData {
 		matches = col.FindAll()
 	}
 
-	var rmi []int
-	for i, c := range matches {
+	// in the most common scenario filteredMatches length will be
+	// the same as matches length, so we avoid allocating per result
+	filteredMatches := make([]types.MatchData, len(matches))
```

Comment:
Think it's good to use `make([], 0, len)` and `append` since it basically allows Go to take care of the `filteredCount` variable while still preallocating

Reply:
done!